src/platforms/javascript/guides/astro/manual-setup.mdx

```diff
@@ -0,0 +1,254 @@
+      debug: false,
+      sourceMapsUploadOptions: {
+        project: "___PROJECT_SLUG___",
+        authToken: process.env.SENTRY_AUTH_TOKEN,
+      },
+    }),
+  ],
+});
+```
+
+If you want to fully customize the browser and client SDKs, e.g. (add a `beforeSend` callback) you can also [manually initialize the SDK](#manual-initialization).
+
+## Manual SDK Initialization
+
+To fully customize the SDK initialization, you can manually initialize the SDK for the client or server side or both.
+At build time, the integration looks for the following two files in the root directory of your config:
+
+- `sentry.client.config.js` containing a `Sentry.init` call for the client side
+- `sentry.server.config.js` containing a `Sentry.init` call for the sever side
+
+These file can also be a `.ts`, `.mjs`, `.mts`, etc. file.
+
+In these files, you can specify the full range of <PlatformLink to="/configuration/options">Sentry SDK options</PlatformLink>.
+
+<Alert level="warning">
+
+If you add a `sentry.client.config.js` or `sentry.server.config.js` file, the options specified in the [Sentry Astro integration](#astro-integration-options) are ignored for the respective runtime.
+The only exception is `sourceMapsUploadOptions` which **always** needs to be set in the integration options.
+
+</Alert>
+
+### Client Side
+
+Here's an example of a custom client-side SDK setup:
+
+```javascript {filename:sentry.client.config.js}
+import * as sentry from "@sentry/astro";
+
+Sentry.init({
+  dsn: "___PUBLIC_DSN___",
+  replaysSessionSampleRate: 0.1,
+  replaysOnErrorSampleRate: 1.0,
+  integrations: [
+    new Sentry.Replay({
+      maskAllText: true,
+      blockAllMedia: true,
+    }),
+    new BrowserTracing({
+      tracePropagationTargets: [/\//, "my-api-domain.com"],
+    }),
+  ],
+  tracesSampler: (samplingContext) => {
+    if (samplingContext.transactionContext.name === "/home") {
+      return 0.5;
+    }
+    return 0.1;
+  },
+});
+```
+
+### Server Side
+
+Here's an example of a custom server-side SDK setup:
+
+```javascript {filename:sentry.server.config.js}
+import * as sentry from "@sentry/astro";
+
+Sentry.init({
+  dsn: "___PUBLIC_DSN___",
+  tracesSampler: (samplingContext) => {
+    if (samplingContext.transactionContext.name === "/home") {
+      return 0.5;
+    }
+    return 0.1;
+  },
+  beforeSend: (event) => {
+    console.log("before send", event);
+    return event;
+  },
+});
+```
+
+### Changing config files location
+
+If you want to change the location of your `sentry.(client|server).config.js` files, specify the path to the files in the Sentry Astro integration options in your `astro.config.mjs`:
+
+```javascript {filename:astro.config.mjs}
+export default defineConfig({
+  // Rest of your Astro project config
+  integrations: [
+    sentryAstro({
+      clientInitPath: ".config/sentryClientInit.js",
+      serverInitPath: ".config/sentryServerInit.js",
+    }),
+  ],
+});
+```
+
+## Configure Source Maps Upload
```

Comment:
Wondering if this content should go in the /platforms/javascript/guides/astro/sourcemaps/ section?

Reply:
Good point. I initially out it into manual setup because that's how we do it in Next and Sveltekit but I agree that it rather belongs to the source maps page. I moved it there and also addressed your concerns about Vite plugin vs. Astro integration/plugin.